prepare/cards/cohere_for_ai.py

```diff
@@ -0,0 +1,74 @@
+from src.unitxt.blocks import (
+    LoadHF,
+    RenameFields,
+    SplitRandomMix,
+    TaskCard,
+)
+from src.unitxt.catalog import add_to_catalog
+from src.unitxt.logging_utils import get_logger
+from src.unitxt.operators import ExecuteExpression, FilterByCondition
+from src.unitxt.settings_utils import get_settings
+from src.unitxt.standard import StandardRecipeWithIndexes
+from src.unitxt.test_utils.card import test_card
+
+settings = get_settings()
+orig_settings = (settings.allow_unverified_code, settings.allow_unverified_code)
+settings.allow_unverified_code = True
+settings.global_loader_limit = 25000  # to ensure language is encountered
```

Comment:
Why is the loader limit necessary?

Reply:
First loading, then filtering by language. If the language that you filter by does not show among the first (that are loaded) your stream is exhausted, throwing exception.
In testing, the limit written in settings overrides the one passed to test_card.